api/pkg/cli/hub/get_resource.go

```diff
@@ -49,6 +59,9 @@ type ResourceResult struct {
 	ResourceContent         *ResourceContent
 }
 
+type AHResourceResult struct {
+}
```

Comment:
wondering if we can start with one command with the use of actual artifacthub api?